src/Packetery/Module/FormFactory.php

```diff
@@ -67,4 +67,24 @@ public function create( ?string $name = null ): Form {
 		$form->allowCrossOrigin();
 		return $form;
 	}
+
+	/**
+	 * Creating a form
+	 *
+	 * @param array $carrierOptions Carrier options.
+	 * @return Form
+	 */
+	public function createCarrierChange( array $carrierOptions ): Form {
```

Comment:
Udělal bych na to raději vlastní třídu. Například Order\CarrierModalFormFactory

Reply:
Trida vytvorena.